agentarchives/atom/client.py

```diff
@@ -42,9 +44,10 @@ class AtomClient(object):
     This change is due to the fact that slugs are visible by users whereas IDs aren't.
     """
 
-    def __init__(self, url, key):
+    def __init__(self, url, key, timeout=None):
         self.key = key
         self.base_url = urljoin(url, 'api/')
+        self.timeout = timeout or DEFAULT_TIMEOUT
```

Comment:
With a timeout parameter of 0, this will use DEFAULT_TIMEOUT - is this intentional?